dev/io.openliberty.microprofile.metrics.internal.5.x.monitor_fat/fat/src/io/openliberty/microprofile/metrics/internal/monitor_fat/MetricsMonitorTest.java

```diff
@@ -0,0 +1,193 @@
+
+import com.ibm.websphere.simplicity.ShrinkHelper;
+import com.ibm.websphere.simplicity.log.Log;
+
+import componenttest.annotation.Server;
+import componenttest.annotation.SkipForRepeat;
+import componenttest.custom.junit.runner.FATRunner;
+import componenttest.topology.impl.LibertyServer;
+
+@RunWith(FATRunner.class)
+public class MetricsMonitorTest {
+
+    private static Class<?> c = MetricsMonitorTest.class;
+
+    @Server("MetricsMonitorServer")
+    public static LibertyServer server;
+
+    @BeforeClass
+    public static void setUp() throws Exception {
+        trustAll();
+    }
+
+    private static void trustAll() throws Exception {
+        try {
+            SSLContext sslContext = SSLContext.getInstance("SSL");
+            sslContext.init(null, new TrustManager[] { new X509TrustManager() {
+                @Override
+                public void checkClientTrusted(X509Certificate[] arg0, String arg1) throws CertificateException {
+                }
+
+                @Override
+                public void checkServerTrusted(X509Certificate[] arg0, String arg1) throws CertificateException {
+                }
+
+                public X509Certificate[] getAcceptedIssuers() {
+                    return null;
+                }
+            } }, new SecureRandom());
+            SSLContext.setDefault(sslContext);
+            HttpsURLConnection.setDefaultSSLSocketFactory(sslContext.getSocketFactory());
+        } catch (Exception e) {
+            Log.error(c, "trustAll", e);
+        }
+    }
+
+    @After
+    public void tearDown() throws Exception {
+        if (server != null && server.isStarted()) {
+            server.stopServer("CWWKS4000E", "CWWKZ0014W");
+            server.removeAllInstalledAppsForValidation();
+        }
+    }
+
+    @Test
+    public void testVendorMetricsPresent() throws Exception {
+
+        String testName = "testDisableMpMetricsFeature";
+
+        Log.info(c, testName,
+                "------- Enable mpMetrics-5.0 vendor metrics should be available ------");
+        server.startServer();
+        Assert.assertNotNull("LTPA keys are not created/ready within timeout period of " + 60000 + "ms.",
+                server.waitForStringInLog("CWWKS4104A.*|CWWKS4105I.*", 60000));
+        Assert.assertNotNull("CWWKO0219I NOT FOUND", server.waitForStringInLog("defaultHttpEndpoint-ssl", 60000));
+        Log.info(c, testName, "------- server started -----");
+        Assert.assertNotNull("CWWKT0016I NOT FOUND", server.waitForStringInLogUsingMark("CWWKT0016I"));
+        checkStrings(getHttpsServlet("/metrics"), new String[] { "mp_scope=\"base\"", "p_scope=\"vendor\"" }, new String[] {});
+
+        Log.info(c, testName, "------- Remove mpMetrics-5.0: no metrics should be available ------");
+        server.setServerConfigurationFile("server_monitorOnly.xml");
+        String logMsg = server.waitForStringInLogUsingMark("CWPMI2003I");
+        
+        //TODO: actually test that /metrics does not respond
```

Comment:
Are you plan to resolve the TODO?

Reply:
Yup, this would be another issue/PR to work on consolidating the FAT with the previous monitor-metric FATS into a singular FAT project.  Also to address this point where the other FATS didn't seem to test that `/metrics` would be disabled when `mpMetrics-x.x` is removed.